### PR TITLE
Remove Spellcheck from Public Key on Account Creation

### LINKED ISSUE #68: Spell check should be turned off for public key field for InterfaceCreateAccount

### PR DESCRIPTION
**Pull request by QuestofIranon**
Closes #68

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -45,7 +45,6 @@ module.exports = {
                     icon: "./build/icons/icon.icns"
                 },
                 linux: {
-                    target: ["pacman", "deb", "rpm"],
```
Comment (mehcode) [inline vue.config.js]: Why did you remove this?

Comment (QuestofIranon) [reply vue.config.js]: Fixed, made the change while on a different branch and I guess it stuck when I switched.
